container_workflow_tool/config/f30.yaml

```diff
@@ -195,8 +194,6 @@ v1:
       git_path: "core"
       git_branch: "fFEDORA"
       user: "pkubat"
-      commands:
-        4: "sed -i 's|registry.fedoraproject.org/fedora:[0-9]*|registry.fedoraproject.org/fedora:30|' */Dockerfile.fedora"
```

Comment:
Would keep these replacement scripts in as we only have one Fedora Dockerfile in upstream and we won to sync to at lest 2 different Fedora releases at any time.

Reply:
Then it should be fedora:31, right?

Reply:
fedora:30, since this config is for fedora 30 and works with f30 branches.